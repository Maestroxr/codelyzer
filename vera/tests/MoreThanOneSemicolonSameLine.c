
int main() 
{
	
	
	/*OK*/
	int i = 0, x = 0;
	{
		;
		;
		{
			;
			;
		}
	}	
	for (i = 0; i <10; i++) 
	{
		x++;
	}
	for (i = 0; i <10;; i++) {x++;}
	/*--------*/
	
	/*PROBLEM*/
	int j = 0;;
	;;
	{
		;;
		{
			;;
		}
	}
	for (i = 0; i <10; i++) 
	{
		x++;;
	}
	for (i = 0; i <10;; i++) {x++;;}
	/*********/
	
}


