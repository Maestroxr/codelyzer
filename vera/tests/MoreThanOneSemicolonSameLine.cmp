C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:24: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:25: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:27: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:29: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:34: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:36: More than one semicolon cannot appear in the same line unless in an if statement.