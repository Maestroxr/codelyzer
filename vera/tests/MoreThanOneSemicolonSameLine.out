C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:20: For loops must have all three loop arguments present.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:24: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:25: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:27: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:29: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:34: More than one semicolon cannot appear in the same line unless in an if statement.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:36: For loops must have all three loop arguments present.
C:\Work\AutoAssessment\codelyzer\vera\tests\MoreThanOneSemicolonSameLine.c:36: More than one semicolon cannot appear in the same line unless in an if statement.
